test/test-network/systemd-networkd-tests.py

```diff
@@ -3481,6 +3491,60 @@ def test_ipv6_proxy_ndp(self):
         for i in range(1, 5):
             self.assertRegex(output, f'2607:5300:203:5215:{i}::1 *proxy')
 
+    def test_ipv6_retrans_time(self):
+        link='test25'
+        copy_network_unit('25-dummy.netdev', '25-dummy.link', copy_dropins=False)
+
+        start_networkd()
+        self.wait_operstate(link, 'off', setup_state='unmanaged')
+        # self.wait_online([f'{link}:degraded'])
+
+        output = check_output(f'ip link show dev {link}')
+        print(output)
```

Comment:
Please drop them, and copy the first .network file and use `wait_online()`.

Reply:
Done.

*NOTE to SELF*: Finding the minimal combination of `.netdev` and `.network` files is what got me unstuck here. Especially, the `.network` file requires at least a `[network]` section with one property configured.